Example_Programs/programming_concpets/ttc_1a/src/c_div.c

```diff
@@ -0,0 +1,19 @@
+#include "header_def.h"
+
+int div(complex_num s1, complex_num s2){
```

Comment:
Passing structures by value is a burden on stack. It may cause stack overflow if the structure size is too big. Use pass by pointer instead.

Reply:
This code can be considered deprecated as I completed and pushed the template complex calculator repository where I used pass by reference(emulated).